example/view/transform.cpp

```diff
@@ -1,26 +1,21 @@
 //! [transform example]
 #include <iostream>
 #include <vector>
-#undef NDEBUG // Force assertions on
-#include <assert.h>
-
 #include <range/v3/view/transform.hpp>
 #include <range/v3/range/conversion.hpp>
 
 int main()
 {
     std::vector<int> numbers{1, 2, 3};
 
-    auto doubled = numbers | ranges::views::transform([](const int& num) { 
-      return num * 2; 
-    }) | ranges::to<std::vector<int>>();
+    auto halved = numbers
+        // Divide each integer by 2, converting it into a double
+        | ranges::views::transform([](const int& num) {
+          return num / 2.0;
+        });
 
-    for (const auto& new_num : doubled) {
-      std::cout << new_num << std::endl;
+    for (const auto& new_num : halved) {
+        std::cout << new_num << std::endl;
     }
```

Comment:
```suggestion
    std::cout << halved << '\n';
```

range-v3 can print views. also [Stop using endl](https://www.youtube.com/watch?v=GMqQOEZYVJQ)

Reply:
Done